Library/Formula/sslsplit.rb

```diff
@@ -0,0 +1,38 @@
+class Sslsplit < Formula
+  homepage "https://www.roe.ch/SSLsplit"
+  url "https://github.com/droe/sslsplit/archive/0.4.11.tar.gz"
+  sha256 "a33808f5d78000d1a919e61bbdfc48478ad40c1a9b09a0250b322f8954cc5ae0"
+
+  depends_on "check" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "libevent"
+  depends_on "openssl"
+
+  # Patch to remove UID 0,GID 0 setting for install
+  patch :DATA
+
+  def install
+    ENV.deparallelize
```

Comment:
Can you try separating into three steps: `make`, `make test`, `make install` and moving the `ENV.deparallelize` around to see which requires it? It may be e.g. that `make` does not but `make test` does.